src/transforms/posthtml.js

```diff
@@ -11,20 +21,28 @@ module.exports = async function(asset) {
   let res = await posthtml(config.plugins).process(asset.ast, config);
 
   asset.ast = res.tree;
-  asset.isAstDirty = true;
-};
+  asset.isAstDirty = true
+}
 
 async function getConfig(asset) {
-  let config = await asset.getConfig(
-    ['.posthtmlrc', '.posthtmlrc.js', 'posthtml.config.js'],
-    {packageKey: 'posthtml'}
-  );
+  let config =
+    asset.getPackage().posthtml ||
```

Comment:
`asset.getPackage()` returns a `Promise` so your expression always evaluate to `undefined` :

https://github.com/parcel-bundler/parcel/blob/16149185bdd386cf478b08629fd2776dce76c943/src/Asset.js#L127-L133

Use `await` :

```js
(await asset.getPackage()).posthtml || ...
```

Reply:
Updated.